platform/ios/src/MGLAPIClient.m

```diff
@@ -87,7 +87,7 @@ - (NSURLRequest *)requestForEvents:(NS_ARRAY_OF(MGLMapboxEventAttributes *) *)ev
 }
 
 - (void)setupBaseURL {
-    NSString *testServerURL = [[NSBundle mainBundle] objectForInfoDictionaryKey:@"MGLMetricsTestServerURL"];
+    NSString *testServerURL = [[NSUserDefaults standardUserDefaults] stringForKey:@"MGLMapboxMetricsTestServerURL"];
```

Comment:
`MGLAPIClientBaseURL` (the production server) is defined as `static NSString * const`, which means a switch to NSURL for `_baseURL` and `testServerURL` wouldn’t make sense here.